src/NoFrixion.MoneyMoov/RestClient/RestApiClient.cs

```diff
@@ -42,8 +42,12 @@ public interface IRestApiClient
     Task<RestApiResponse> PutAsync(string path, string accessToken);
 
     Task<RestApiResponse<T>> PutAsync<T>(string path, string accessToken, HttpContent content);
+    
+    Task<RestApiResponse<T>> PutAsync<T>(string path, string accessToken, HttpContent content, string rowVersion);
 

```

Comment:
Is rowVersion required or should it be nullable or default to null. So it's not required for all calls.